examples/mcp-servers/custom-bundle-dir.json

```diff
@@ -0,0 +1,13 @@
+{
+  "mcpServers": {
+    "troubleshoot": {
+      "command": "docker",
+      "args": [
+        "run",
```

Comment:
All examples should include the "--rm" flag I'm only commenting on thsi one but add it to all examlpes that you're adding to this folder.